addons/point_of_sale/static/src/xml/pos.xml

```diff
@@ -74,17 +74,20 @@
 
     <t t-name="ProxyStatusWidget">
         <div class="oe_status js_proxy">
-            <span class='js_msg oe_orange oe_hidden'></span>
-            <div class="js_connected oe_icon oe_green">
+            <div class="js_connected oe_green">
+                <span class='js_msg oe_hidden'></span>
```

Comment:
can you double quote the one all your 'js_msg oe_hidden', it will be beautifful in futurewhen all lines will be changed